src/components/admin/form-components/SearchableSelector.jsx

```diff
@@ -0,0 +1,131 @@
+/* eslint react/jsx-no-bind: 0 */
```

Comment:
This shouldn't be necessary should it?

Reply:
This was left in from the original implementation of the component, but yeah whatever made it necessary when whoever wrote it is now refactored - TODO

Reply:
Was probably me while I was ignorant haha